src/app/layout.tsx

```diff
@@ -31,18 +32,20 @@ export default function RootLayout({
       </head>
       <body>
         <DatasetBasketProvider>
-          <div className="grid h-screen w-full grid-rows-[auto_1fr_auto]">
-            <SessionProviderWrapper>
-              <div>
-                <Header />
-              </div>
-              <div>{children}</div>
-              <div>
-                <Disclaimer />
-                <Footer />
-              </div>
-            </SessionProviderWrapper>
-          </div>
+          <ApplicationProvider>
```

Comment:
Actually ApplicationProvider is only seems to be used accross Applications page. Does it make sense to move this provider inside the applications page component instead of it globally?